tests/prf/test_tpfmodel.py

```diff
@@ -7,20 +7,28 @@
 from numpy.testing import assert_allclose
 from scipy.stats import mode
 
-from lightkurve.prf import FixedValuePrior, GaussianPrior, UniformPrior
-from lightkurve.prf import StarPrior, BackgroundPrior, FocusPrior, MotionPrior
-from lightkurve.prf import TPFModel, PRFPhotometry
 from lightkurve.prf import SimpleKeplerPRF, KeplerPRF
 
+try:
+    from oktopus import PoissonPosterior
+    from lightkurve.prf import FixedValuePrior, GaussianPrior, UniformPrior
+    from lightkurve.prf import StarPrior, BackgroundPrior, FocusPrior, MotionPrior
+    from lightkurve.prf import TPFModel, PRFPhotometry
+    NO_OKTOPUS = False
+except ModuleNotFoundError:
```

Comment:
I would also suggest adding a comment here that octopus is a required dependency for tpfmodels only, and is not supported by numpy2.x